src/jupyter_contrib_nbextensions/nbextensions/notify/notify.js

```diff
@@ -8,9 +8,18 @@ Add this file to $(ipython locate)/nbextensions/
 
 */
 
-define(["require"], function (require) {
+define([
+  "jquery",
+  "base/js/namespace",
+  "require",
+], function ($, IPython, require) {
```

Comment:
It makes more sense to assign the namespace to the `Jupyter` variable, since that's what it's now called, and what you've used later on in the file. For now, both work since both IPython and Jupyter are defined in the global namespace, but it's less confusing for people reading this file if we stick to the new namespace which we definitely get from the `define` call...